be/src/storage/page_cache.h

```diff
@@ -36,8 +36,8 @@ namespace starrocks {
 class PageCacheHandle;
 class MemTracker;
 
-// Page cache min size is 512MB
-static constexpr int64_t kcacheMinSize = 536870912;
+// Page cache min size is 256MB
+static constexpr int64_t kcacheMinSize = 268435456;
```

Comment:
why changed to 256M?

Reply:
The previous policy was to disable pagecache if the user set the pagecache size to less than 512MB. That is, config::disable_storage_page_cache was set to true. Because we currently do not support dynamic Pagecache on or off, when we introduce the dynamic change Pagecache size feature, if the user increases the Pagecache size, how to deal with the previously closed Pagecache. Therefore, to be conservative, if the user has Pagecache enabled, the cache size should be >= 256MB.